react-components/src/metadata/MetadataList.js

```diff
@@ -0,0 +1,295 @@
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import { withStyles } from "@material-ui/core/styles";
+
+import ContentAdd from "@material-ui/icons/Add";
+import ContentRemove from "@material-ui/icons/Delete";
+import ContentEdit from "@material-ui/icons/Edit";
+import ContentView from "@material-ui/icons/List";
+
+class MetadataGridToolbar extends Component {
+    static propTypes = {
+        onAddAVU: PropTypes.func.isRequired,
+    };
+
+    render() {
+        const { parentID, editable, classes, intl, onAddAVU } = this.props;
+
+        return (
+            <Toolbar
+                className={classes.root}
+            >
+                {editable &&
+                <div className={classes.actions}>
+                    <Fab id={build(parentID, ids.BUTTONS.ADD)}
+                         size="small"
+                         color="primary"
+                         aria-label={formatMessage(intl, "addMetadata")}
+                         onClick={onAddAVU}
+                    >
+                        <ContentAdd/>
+                    </Fab>
+                </div>}
+                <div className={classes.title}>
+                    <Typography id={build(parentID, ids.TITLE)} variant="h6">
+                        {getMessage("avus")}
+                    </Typography>
+                </div>
+            </Toolbar>
+        );
+    }
+}
+
+MetadataGridToolbar = withStyles(styles)(withI18N(injectIntl(MetadataGridToolbar), intlData));
+
+const columnData = [
+    {
+        id: build(ids.COL_HEADER, ids.AVU_ATTR),
+        key: "attr",
+        name: getMessage("attribute"),
+        enableSorting: true,
+    },
+    {
+        id: build(ids.COL_HEADER, ids.AVU_VALUE),
+        key: "value",
+        name: getMessage("value"),
+        enableSorting: true,
+    },
+    {
+        id: build(ids.COL_HEADER, ids.AVU_UNIT),
+        key: "unit",
+        name: getMessage("metadataUnitLabel"),
+        enableSorting: true,
+    },
+    {
+        id: build(ids.COL_HEADER, ids.AVU_AVUS),
+        key: "avus",
+        name: getMessage("metadataChildrenLabel"),
+        enableSorting: true,
+        numeric: true,
+        padding: "none",
+    },
+    {
+        id: build(ids.COL_HEADER, ids.COL_ACTIONS),
+        key: "actions",
+        enableSorting: false,
+        padding: "none",
+    },
+];
+
+class MetadataList extends Component {
+    constructor(props) {
+        super(props);
+
+        this.newAttrCount = 1;
+
+        this.state = {
+            orderBy: props.orderBy,
+            order: props.order || "asc",
+        };
+
+        [
+            "onAddAVU",
+            "handleRequestSort",
+        ].forEach(methodName => (this[methodName] = this[methodName].bind(this)));
+    }
+
+    newAttrName() {
+        return `New attribute ${this.newAttrCount++}`;
```

Comment:
Could this be i18n-ized?

Reply:
Good catch. Looks like I already created an `intl` message for use in `EditAttributeFormList`, but I forgot to come back to this component and add it here too 😆 :
https://github.com/cyverse-de/ui/blob/7bc4e878f27fd90295a0ef0aad5f88aee3aae52e/react-components/src/metadata/admin/EditAttributeFormList.js#L288-L290